Cannot build on Ubuntu 20.10

# System info

```
> lsb_release -a
Distributor ID:	Pop
Description:	Pop!_OS 20.10
Release:	20.10
Codename:	groovy
```

```
> uname -a
Linux daniel-darter 5.11.0-7614-generic #15~1618626693~20.10~ecb25cd-Ubuntu SMP Thu Apr 22 16:00:45 UTC  x86_64 x86_64 x86_64 GNU/Linux
```

Reproduced on lldb-sys.rs HEAD, v0.0.21, and v0.0.20. Didn't check older.

# To reproduce

```bash
> gh repo clone endoli/lldb-sys.rs
> cd lldb-sys.rs
> cargo build
```

# The error

```
src/lldb/Bindings/SBCommandInterpreterBinding.cpp: In function ‘SBCommandInterpreterRunOptionsOpaque* CreateSBCommandInterpreterRunOptions()’:
  src/lldb/Bindings/SBCommandInterpreterBinding.cpp:22:99: error: invalid use of incomplete type ‘class lldb::SBCommandInterpreterRunOptions’
     22 |     return reinterpret_cast<SBCommandInterpreterRunOptionsRef>(new SBCommandInterpreterRunOptions());
        |                                                                                                   ^
  In file included from /usr/lib/llvm-11/include/lldb/API/SBBreakpoint.h:12,
                   from src/lldb/Bindings/SBBreakpointBinding.h:15,
                   from src/lldb/Bindings/LLDBBinding.h:21,
                   from src/lldb/Bindings/SBCommandInterpreterBinding.cpp:10:
```
([full stderr](https://gist.github.com/danielzfranklin/d9c3bd31e46961f730215ed82a360fee))

--- a/src/lldb/Bindings/SBCommandInterpreterBinding.cpp
+++ b/src/lldb/Bindings/SBCommandInterpreterBinding.cpp
@@ -9,6 +9,7 @@
 
 #include "lldb/Bindings/LLDBBinding.h"
 #include "lldb/API/LLDB.h"
+#include "lldb/API/SBCommandInterpreterRunOptions.h"
 
 using namespace lldb;
 

